frontends/ol-components/src/components/LearningResourceExpanded/LearningResourceExpanded.tsx

```diff
@@ -0,0 +1,425 @@
+        url={resource?.url}
+        embedlyKey={config.key}
+      />
+    )
+  } else if (resource?.image) {
+    return (
+      <Image
+        src={resourceThumbnailSrc(resource?.image, config)}
+        aspectRatio={config.width / config.height}
+        alt={resource?.image.alt ?? ""}
+      />
+    )
+  } else {
+    return <SkeletonImage variant="rectangular" height={220} />
+  }
+}
+
+const ResourceTitle = ({ resource }: { resource?: LearningResource }) => {
+  if (!resource) {
+    return <Skeleton variant="text" height={20} width="66%" />
+  }
+  return (
+    <Typography variant="h5" component="h2">
+      {resource.title}
+    </Typography>
+  )
+}
+
+const ResourceDescription = ({ resource }: { resource?: LearningResource }) => {
+  if (!resource) {
+    return (
+      <>
+        <Skeleton variant="text" width="100%" />
+        <Skeleton variant="text" width="100%" />
+        <Skeleton variant="text" width="100%" />
+        <Skeleton variant="text" width="100%" />
+        <Skeleton variant="text" width="100%" />
+        <Skeleton variant="text" width="33%" />
+      </>
+    )
+  }
+  return <Description>{resource.description}</Description>
+}
+
+const TopicsSection: React.FC<{ topics: LearningResourceTopic[] }> = ({
+  topics,
+}) => {
+  return (
+    <Topics>
+      <Typography variant="subtitle2" component="h3">
+        Topics
+      </Typography>
+      <TopicsList>
+        {topics.map((topic) =>
+          topic.channel_url ? (
+            <ChipLink
+              key={topic.id}
+              size="medium"
+              label={topic.name}
+              href={topic.channel_url}
+              variant="outlined"
+            />
+          ) : (
+            <Chip
+              key={topic.id}
+              size="medium"
+              label={topic.name}
+              variant="outlined"
+            />
+          ),
+        )}
+      </TopicsList>
+    </Topics>
+  )
+}
+
+const InfoSection = ({ run }: { run?: LearningResourceRun }) => {
+  if (!run) {
+    return null
+  }
+  return (
+    <InfoItems>
+      <Typography variant="subtitle2" component="h3">
+        Info
+      </Typography>
+      <InfoItem>
+        <RiMoneyDollarCircleFill />
+        <InfoLabel>Cost:</InfoLabel>
+        <InfoValue>{run.prices?.[0] || "-"}</InfoValue>
+      </InfoItem>
+      <InfoItem>
+        <RiBarChartFill />
+        <InfoLabel>Level:</InfoLabel>
+        <InfoValue>{run.level?.[0]?.name || "-"}</InfoValue>
+      </InfoItem>
+      <InfoItem>
+        <RiGraduationCapFill />
+        <InfoLabel>Instructors:</InfoLabel>
+        <InfoValue>
+          {run.instructors?.length
+            ? run.instructors.map(({ full_name: name }) => name).join(", ")
+            : "-"}
+        </InfoValue>
+      </InfoItem>
+      <InfoItem>
+        <RiGlobalLine />
+        <InfoLabel>Languages:</InfoLabel>
+        <InfoValue>
+          {run.languages?.length ? run.languages.join(", ") : "-"}
```

Comment:
These are returned on the API in ISO language / region codes (e.g. `en-us`). Do we have a map? We could use this https://github.com/mozilla/language-mapping-list (though they capitalize the region, `en-US`). Ideally the human readable form would come from the API.

Reply:
https://www.npmjs.com/package/iso-639-1 is already installed in the repo. I would use that.